src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -743,7 +743,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         // Use this map to discard duplicates.
         std::unordered_map<std::wstring_view, bool> foundCommands{};
 
-        auto backspaces = std::wstring(::base::saturated_cast<size_t>(0), L'\x7f');
+        auto backspaces = std::wstring(currentCommandline.size(), L'\x7f');

```

Comment:
Some thoughts:
* Why did this change in this PR in particular?
* Why does it say "backspaces" but uses DELs? Shouldn't it use `\b`?
* `fmt::format` below should use `FMT_COMPILE` if possible.
* The proper way to do this is to use `{:\x7f^{}}` with `""` and `currentCommandline.size()` as the arguments. This will pad the empty string to be at least that many padding characters wide. (For `printf()` functions you'd use the same technique.) Here's an example: https://godbolt.org/z/Gd5hfaqKj
* It would probably help with readability if you put the `fmt::format` call on an extra line.
* Since `Model::SendInputArgs` is a WinRT projection, shouldn't the argument technically be a `winrt::param::hstring` and thus accept a `std::wstring_view` as the argument? In that case you can drop the `winrt::hstring{}` wrapping.